docs/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -168,7 +168,7 @@ for the application, extend the configuration of the module.
 Add the following lines to the module configuration file, e.g.
 `module/Album/config/module.config.php`:
 
-<pre class="language-php" data-line="6-7"><code>
+```php
```

Comment:
Please do not remove, otherwise the line highlights will be gone.

Reply:
Oh, I only now see the purpose of `data-line="6-7"`  https://docs.laminas.dev/laminas-form/application-integration/usage-in-a-laminas-mvc-application/#register-form-and-controller sorry ^^

Reply:
@Slamdunk 
Oh no, that's not your fault. No [other syntax](https://python-markdown.github.io/extensions/attr_list/) works here.